pyaedt/hfss.py

```diff
@@ -6217,39 +6254,39 @@ def wave_port(
          :class:`pyaedt.modeler.elements3d.FacePrimitive`
             Main object for port creation or starting object for the integration line.
         reference : int, str, list or :class:`pyaedt.modeler.cad.object3d.Object3d`
-            Ending object for the integration line or reference for Terminal solution. Can be multiple objects.
+            The ending object for the integration line or reference for Terminal solution. Can be multiple objects.
```

Comment:
```suggestion
            Ending object for the integration line or reference for Terminal solution. Can be multiple objects.
```
In the initial brief description, we drop the article (An, A, The) so that the description begins with an important word rather than with an article.